scripts/create-cli-demo.sh

```diff
@@ -16,9 +16,10 @@ wgc subgraph create family --label team=A env=production --routing-url http://fa
 wgc subgraph create hobbies --label team=B env=production --routing-url http://hobbies:4003/graphql
 wgc subgraph create products --label team=B env=production --routing-url http://products:4004/graphql
 
-wgc subgraph publish employees --schema ../demo/employees/subgraph/schema.graphqls
-wgc subgraph publish family --schema ../demo/family/subgraph/schema.graphqls
-wgc subgraph publish hobbies --schema ../demo/hobbies/subgraph/schema.graphqls
-wgc subgraph publish products --schema ../demo/products/subgraph/schema.graphqls
+SUBGRAPHS="employees family hobbies products"
 
-wgc router token create mytoken --graph-name production
\ No newline at end of file
+for subgraph in $SUBGRAPHS; do
+  pnpm wgc subgraph publish $subgraph --schema demo/pkg/subgraphs/$subgraph/subgraph/schema.graphqls
```

Comment:
Just `wgc`

Reply:
That was fast! I amended it like 30 after opening it 😅

Reply:
Yeah both happened at the same time XD